eva-accession-import-dbsnp2/src/test/java/uk/ac/ebi/eva/accession/dbsnp2/io/DbsnpJsonItemReaderTest.java

```diff
@@ -0,0 +1,81 @@
+package uk.ac.ebi.eva.accession.dbsnp2.io;
+
+import org.apache.commons.lang3.StringUtils;
+import org.junit.After;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.batch.item.ExecutionContext;
+import org.springframework.batch.item.ItemStreamException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+import uk.ac.ebi.eva.accession.dbsnp2.parameters.InputParameters;
+import uk.ac.ebi.eva.accession.dbsnp2.test.BatchTestConfiguration;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(SpringRunner.class)
+@ContextConfiguration(classes = {BatchTestConfiguration.class})
+@TestPropertySource({"classpath:application.properties"})
+public class DbsnpJsonItemReaderTest {
+
+    private String sourceFile = StringUtils.EMPTY;
```

Comment:
Using a library to create an empty string is overkill, specially when the value of the attribute is set in every test.
```suggestion
    private String sourceFile;
```